plugins/modules/proxmox.py

```diff
@@ -729,7 +731,9 @@ def get_ansible_module():
         ],
         mutually_exclusive=[
             # Creating a new container is done either by cloning an existing one, or based on a template.
-            ("clone", "ostemplate", "update"),
+            ("clone", "ostemplate"),
+            ("clone", "update"),
+            ("force", "update"),
```

Comment:
The entry `("force", "update")` is a breaking change BTW - right now you can explicitly say `force=True` and `update=True`, or `force=True` and `update=False` - but you won't be able to do that anymore.

I'm not sure we should really have this here.

Reply:
I added 475ff6fe7fa5743a0f6dd0635c500687896eddb5 which at least documents this. I guess this is fine, but I'm not using the module, so :shrug: